app/controllers/services_controller.rb

```diff
@@ -10,18 +10,22 @@ def authorize!
 
   def destroy
     service_id = params[:id]
-    discontinued_service = DiscontinuedService.new({
-      service_id: service_id,
-      recorded_by_educator_id: current_educator.id,
-      discontinued_at: Time.now
-    })
-
-    serializer = ServiceSerializer.new(Service.find(service_id))
-
-    if discontinued_service.save
-      render json: serializer.serialize_service
+    if Service.find(service_id).date_started > Time.now
```

Comment:
Basically I am destroying the service completely if its a future scenario. This is where the discontinued button eventually points to.